Content.Shared/Preferences/PlayerPreferences.cs

```diff
@@ -47,7 +48,7 @@ public ICharacterProfile GetProfile(int index)
 
         public int FirstEmptySlot()
         {
-            var firstEmpty = IndexOfCharacter(null);
+            var firstEmpty = IndexOfCharacter(null!);
```

Comment:
null! is suss and this should *probably* be handled?

Reply:
This method isn't even used...